analysis_options.yaml

```diff
@@ -143,7 +143,7 @@ dart_code_metrics:
     - test/**
   metrics:
     number-of-arguments: 8
-    number-of-methods: 28
+    number-of-methods: 32
```

Comment:
```
doc/examples/particles/lib/main.dart:
WARNING MyGame - number of methods: 30
```

again, I don't see any particular value in this metric :shrug: 
I could break down the particles example but I don't think that is necessarily better.